client/src/atoms/Toogle/Toggle.tsx

```diff
@@ -12,6 +12,7 @@ import {
   SwitchTextStyled
 } from './style'
 import { RequiredStyled } from 'organisms/Form/styles'
+import { handleFormColor } from 'organisms/Form/base';
```

Comment:
This is an independent atom and should not rely on anything outside it's own file structure